# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/windows/ia2/ia2AccessibleValue.cpp
# Commit: dcc478201cb7
# Full Hash: dcc478201cb75d078d3a5fd33f306c7b9cfcd527
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2021-08-27 03:42:22
# Regressor Bug: 1727643
# File Overlap Count: 1
# Description:
#   Bug 1727643 - P4: Make use of new common methods in platforms and XPCOM. r=morgan
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123698
# ==============================================================================

diff -r 1a9901750064 -r dcc478201cb7 accessible/windows/ia2/ia2AccessibleValue.cpp
--- a/accessible/windows/ia2/ia2AccessibleValue.cpp	Thu Aug 26 19:40:05 2021 +0000
+++ b/accessible/windows/ia2/ia2AccessibleValue.cpp	Thu Aug 26 19:40:06 2021 +0000
@@ -21,6 +21,10 @@
   return static_cast<MsaaAccessible*>(this)->LocalAcc();
 }
 
+Accessible* ia2AccessibleValue::Acc() {
+  return static_cast<MsaaAccessible*>(this)->Acc();
+}
+
 // IUnknown
 
 STDMETHODIMP
@@ -51,7 +55,7 @@
 
   VariantInit(aCurrentValue);
 
-  AccessibleWrap* valueAcc = LocalAcc();
+  Accessible* valueAcc = Acc();
   if (!valueAcc) {
     return CO_E_OBJNOTCONNECTED;
   }
@@ -84,7 +88,7 @@
 
   VariantInit(aMaximumValue);
 
-  AccessibleWrap* valueAcc = LocalAcc();
+  Accessible* valueAcc = Acc();
   if (!valueAcc) {
     return CO_E_OBJNOTCONNECTED;
   }
@@ -105,7 +109,7 @@
 
   VariantInit(aMinimumValue);
 
-  AccessibleWrap* valueAcc = LocalAcc();
+  Accessible* valueAcc = Acc();
   if (!valueAcc) {
     return CO_E_OBJNOTCONNECTED;
   }